test/test-commands.js

```diff
@@ -420,3 +420,27 @@ describe("toggleHeaderColumn", () => {
           toggleHeaderColumn,
           doc(table(tr(c11, c11), tr(c11, c11)))))
 })
+
+describe("toggleHeader", () => {
+  it("turns a header row with colspan and rowspan into a regular cell", () =>
+    test(doc(p('x'), table(tr(h(2, 1), h(1, 2)), tr(cCursor, c11), tr(c11, c11, c11))),
+         toggleHeader("row", { useDeprecateLogic: false }),
```

Comment:
useDeprecateLogic is `false` by default, there's no need to pass it here

Reply:
I wanted to be really explicit on this one